### DISCUSSION TITLE
When I choose the theme, how do I put the card in my README?

### DISCUSSION THREAD

**Discussion by CondratKenasil** [anuraghazra/github-readme-stats#2290] [Q&A] [answered]
When I choose the theme, how do I put the card in my README?

**Comment by MitkoVtori** [comment, marked as answer]
Hi, just copy-paste the link with the theme you wanted:
![Screenshot 2022-11-24 122512](https://user-images.githubusercontent.com/112943652/203760772-e21e7676-f81d-44e3-98cc-f5a99208045c.png)

**Reply by CondratKenasil** [reply]
Thanks

**Reply by rickstaa** [reply]
@MitkoVtori Thank you for jumping on this one! 🚀 One small GitHub tip you can add code blocks so that people can click the copy button (see https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks).

```md
[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&theme=dark)](https://github.com/anuraghazra/github-readme-stats)
```

[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&theme=dark)](https://github.com/anuraghazra/github-readme-stats)
